src/Build/ElementLocation/XmlDocumentWithLocation.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.IO;
 using System.Xml;
+using Microsoft.Build.Framework;
```

Comment:
Necessary?

Reply:
Yes, that is for change wave check.

Reply:
Ah, wait, I moved the code, no more necessary!